eiffel-vocabulary/EiffelServiceDeployedEvent.md

```diff
@@ -20,7 +20,41 @@ The EiffelServiceDeployedEvent is to declare when a configuration item/VM  and/o
 It implies that the service is not in a running state and is ready to be started.
 
 ## Data Members
- 
+### data.servicename
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the Service that was deployed.
+
+### data.instancename
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The name of the instance that was deployed.
+
+### data.owner
+__Type:__ String  
+__Required:__ Yes  
+__Description:__ The user that owns the Service instance. 
```

Comment:
Is it the execution user that runs the instance, or the responsible human behind this?

Reply:
Same as above...

The "user" is a person or a machine identity.

Proposal to change Description to :
The entity that has been assigned/responsible for the Service